api/integrations/amplitude/constants.py

```diff
@@ -0,0 +1 @@
+DEFAULT_AMPLITUDE_API_URL = "https://api.amplitude.com"
```

Comment:
```suggestion
DEFAULT_AMPLITUDE_API_URL = "https://api2.amplitude.com"
```

https://www.docs.developers.amplitude.com/analytics/apis/identify-api/#endpoints

Reply:
Ah, interesting, looks like that has changed since we originally wrote this code as well. 

I'll need to verify that there is no issue for existing customer integrations changing their URLs from api.amplitude.com (as it currently is) to api2.amplitude.com.